lib/mountain_view/presenter.rb

```diff
@@ -0,0 +1,70 @@
+module MountainView
+  class Presenter
+    class_attribute :_attributes, instance_accessor: false
+    self._attributes = {}
+
+    attr_reader :slug, :properties
+    alias_method :props, :properties
+
+    def initialize(slug, properties={})
+      @slug = slug
+      @properties = attribute_defaults.deep_merge(properties)
+    end
+
+    def partial
+      "#{slug}/#{slug}"
+    end
```

Comment:
Ah, unfortunately Rails does something a bit special with `to_partial_path`. If the controller is namespaced (such as `MountainView::StyleguideController`) then it'll namespace the `to_partial_path` with the controller namespace, resulting in `mountain_view/component/component/`. See the following method..

https://github.com/rails/rails/blob/20559834e4959531f02f85cac433127c2cc52b20/actionview/lib/action_view/renderer/partial_renderer.rb#L482

The only way I can see to prevent this behaviour would be to override `partial_path` entirely. Not awful, but maybe overkill for what we're trying to achieve.